feat(integrations): add support for tripletex (#6300)
NangoHQ/nango: docs/api-integrations/tripletex/connect.mdx

---
title: Tripletex - How do I link my account?
sidebarTitle: Tripletex
---

# Overview

To authenticate with Tripletex you need:

1. **Consumer Token** — provided to you by the developer of the integration. It identifies the integration, not your account.
2. **Employee Token** — generated by you inside your Tripletex tenant. It controls which data the integration can read or write on your behalf.

> This guide walks you through generating the employee token and pasting both values into Nango. Nango handles the session-token minting and refresh in the background.

Reviewer: ```suggestion
This guide walks you through generating the employee token within Tripletex.
```

Author: Done 6eaa2a340 8927bd52f

### Prerequisites

> - A Tripletex production account (`tripletex.no`). For testing, use the [Tripletex (Sandbox)](/api-integrations/tripletex-sandbox/connect) integration instead.

Reviewer: Since this will be merged into a single provider, could you also mention in the docs if there is a distinguishable step compared to the sandbox version?

Author: Added a note in 7043838f5. The flow's basically identical, the one real difference is how you get the consumer token: in the test env you self-serve it instantly from a free account, but for prod you have to apply and get Tripletex to approve it. It's a screening that's mandatory even for internal integrations, takes ~2-3 weeks, and the consumer token + app name come back in the approval email. See: https://developer.tripletex.no/docs/documentation/faq/getting-started/#step-3-getting-ready-for-production

<img src="/api-integrations/tripletex/04-manual-setup.png" />

7. Tripletex groups entitlements by module (general access, time sheet, invoicing, accounting, salary, and more). Enable only the scopes the integration needs.

<img src="/api-integrations/tripletex/05-entitlements.png" />

8. Give the token a memorable name and click **Create token**.

<img src="/api-integrations/tripletex/06-new-token-filled.png" />

   <Warning>If you see *"The application could not be found"*, the application name does not match what the developer registered with Tripletex. The name is case-sensitive and must match exactly.</Warning>

9. Tripletex shows the generated employee token in a one-time popup. **Copy it immediately** — it cannot be recovered. If you misplace it, generate a new one (which invalidates the old one).

<a id="step-3-set-the-proxy-company-id"></a>
#### Step 3: Set the proxy company ID

Leave this at `0` unless the integration developer specifically told you otherwise. A non-zero value is only required for accountant-token flows where one token spans multiple client tenants.

<a id="step-4-submit-the-connect-form"></a>

> #### Step 4: Submit the connect form

Reviewer: For this step, it would be great to have the same style as the other docs.

Author: Done 3597abc25
